src/index.js.flow

```diff
@@ -419,6 +419,11 @@ export type DAOConfig<TEntity, TEntityMutator> = {
 };
 
 declare export class DAOResult<TEntity> {
+  constructor(
```

Comment:
`|` is working okey there though... I'm about the best way annotate class constructor in general:
I made it `constructor(args: Args): void`, but in immutableJS i saw `static (args: Args): ClassName`..may be there are some more patterns.